docs/controls/InAppNotification.md

```diff
@@ -0,0 +1,129 @@
+# In App Notification 
+
+The *In App Notification* control offers the ability to show local notifications in your application.
+
+## Syntax
+
+The control should be placed where you want your notification to be displayed in the page, generally in the root grid.
```

Comment:
@Vijay-Nirmal By default, the position is set to Bottom because I mimic the original component from MS Edge. But of course, you can place it at the Top or at the Center, it depends on the dev intention.